### PR TITLE
Issue/DW-27

### PR DESCRIPTION
**Pull request by bitcoinbrisbane**
Add awesome new mesh members.

### CODE REVIEW

**Review by RalphLavelle — APPROVED** [review]

File: the/mesh.html [hunk 1]
```diff
@@ -108,7 +108,61 @@ <h2 class="to-animate">Meet the Mesh</h2>
...
+                  <img src="../images/The-Mesh/Ralph-Lavelle.jpg" alt="Ralph" />
+                </figure>
+                <div class="div-text">
+                  <p class="mesh-name">Ralph</p>
+                  <p class="mesh-title">Sudo Developer</p>
+                </div>
+              </div>
+              <div class="grid-element">
+                <figure class="circular-landscape">
+                  <img src="../images/The-Mesh/Barry-Easman.jpg" alt="Barry" />
```
Comment (bazmatic) [inline the/mesh.html:120]: File should be `Barry-Earsman.jpg`

File: the/mesh.html [hunk 2]
```diff
@@ -108,7 +108,61 @@ <h2 class="to-animate">Meet the Mesh</h2>
...
+                  <p class="mesh-name">Preston</p>
+                  <p class="mesh-title">Developer</p>
+                </div>
+              </div>
+              <div class="grid-element">
+                <figure class="circular-landscape">
+                  <img src="../images/The-Mesh/Taylor-Scheel.jpg" alt="Taylor" />
+                </figure>
+                <div class="div-text">
+                  <p class="mesh-name">Taylor Scheel</p>
```
Comment (bazmatic) [inline the/mesh.html:141]: Some members have just first name, others have both first and last names. Any reasoning behind that?

File: index.html [hunk 3]
```diff
@@ -82,7 +82,7 @@ <h2 style="word-break: break-all" class="to-animate">
                 <a href="./clients.html" class="">CLIENTS</a>
                 <a href="https://opencollective.com/dltxio" target="_blank">OPEN SOURCE</a>
                 <a href="./agile.html">AGILE</a>
-                <a href="./themesh.html">THE MESH</a>
+                <a href="the/mesh.html">THE MESH</a>
```
Comment (bazmatic) [inline index.html:85]: Initial directory context "." has been left out here. Better to keep this consistent. I like the "the" though :-)

File: the/mesh.html (hunk 2 above)
Comment (bitcoinbrisbane) [reply the/mesh.html:151]: Resolved

File: index.html (hunk 3 above)
Comment (bitcoinbrisbane) [reply index.html:85]: Webstorm did it, but good catch.

**Pull request merged**